package.json

```diff
@@ -7,9 +7,8 @@
     "dev": "vite dev"
   },
   "dependencies": {
-    "instant-noodles": "^0.0.3",
-    "react": "<18.0.0",
-    "react-dom": "<18.0.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@testing-library/react": "12.1.4",
```

Comment:
Looks good! Can we also include removing the `@testing-library/react` with these updates?

Reply:
Saw the new updates in the other PR(https://github.com/codepath/twitter_starter/pull/5) you opened, thank you 👍